### ISSUE TITLE
Language name is Go

### ISSUE THREAD

**Issue by AlekSi** [luciotato/golang-notes#15]
"golang" is used for tags, for SEO, etc. Please use "Go" for the actual name in text.

**Comment by jeremejazz** [comment]
Good point. Though it would be difficult to change the repository name/url since it already has been indexed Google search. Perhaps just the description and page titles.
